src/routes/(app)/settings/profile/+page.svelte

```diff
@@ -23,24 +37,197 @@ $effect(() => {
             updateAccount: false,
         }).then((response) => {
             metadata = response?.metadata ?? {};
+            displayName = metadata.display_name ?? "";
+            name = metadata.name ?? "";
+            about = metadata.about ?? "";
+            website = metadata.website ?? "";
+            nip05 = metadata.nip05 ?? "";
+            lud16 = metadata.lud16 ?? "";
+            banner = metadata.banner ?? "";
+            picture = metadata.picture ?? "";
         });
     }
 });
 
+$inspect(metadata);
+
+async function uploadImageToNip96(file: File): Promise<string> {
```

Comment:
I think this function is too large, and also doesn't seem to be directly related to the component. Is there any reason why this should be handled by the client instead of in rust? Regardless of that, I'd move it to a "Nip96Client" class with an "upload_image" method. What do you think?